src/tests/conformance/query/ping_pong.rs

```diff
@@ -3,19 +3,27 @@
 //!
 //!     - mtPING (with PingType::PtPing) -> mtPING (with PingType::PtPong)
 
+use std::time::Duration;
+
 use rand::{thread_rng, RngCore};
+use tempfile::TempDir;
+use tokio::time::{sleep, Instant};
 
 use crate::{
     protocol::{
         codecs::message::{BinaryMessage, Payload},
         proto::{tm_ping::PingType, TmPing},
     },
+    setup::node::{Node, NodeType},
     tests::conformance::{perform_expected_message_test, TestConfig},
+    tools::synth_node::SyntheticNode,
 };
 
+const EXPECTED_PING_MESSAGE_TIMEOUT: Duration = Duration::from_secs(70);
```

Comment:
since the ping message is expected every 60 secs, let's try to make test duration shorter if possible 
```suggestion
const EXPECTED_PING_MESSAGE_TIMEOUT: Duration = Duration::from_secs(62);
```

Reply:
Done